eo-maven-plugin/src/test/java/org/eolang/maven/VerifyMojoTest.java

```diff
@@ -73,10 +74,47 @@ void detectsErrorsSuccessfully(@TempDir final Path temp) {
                 .execute(new FakeMaven.Verify()),
             "Program with noname attributes should have failed or error, but it didn't"
         );
+        Assertions.assertTrue(
+            out.captured().stream().anyMatch(
+                log -> log.contains(
+                    "Errors identified"
```

Comment:
@Yanich96 the main point of the ticket is to check if exception contains sufficient and detailed information about an error. In most cases filename and error line is enough. In your test you checks that log contains "Error identified" string. What such information can tell us about the error? Nothing specific. 
Specifics would be a filename and line where error appears. So you need to check if log contains valid filename and line with an error. The same story with all other tests you made.

Reply:
@Yanich96 if you take a look inside `fail-on-critical.xsl` you may see the next code:
```
<xsl:text>Critical error identified:
</xsl:text>
      <xsl:for-each select="/program/errors/error[@severity='critical']">
  <xsl:value-of select="concat('  ', /program/@source, ', ', @line, ': ',  text() , ';&#10;')"/>
</xsl:for-each>
```
Here file name and line is printed